src/Compute/Compute/Generated/VirtualMachineScaleSet/Config/NewAzureRmVmssConfigCommand.cs

```diff
@@ -241,6 +241,10 @@ public partial class NewAzureRmVmssConfigCommand : Microsoft.Azure.Commands.Reso
             ValueFromPipelineByPropertyName = true)]
         public string[] IdentityId { get; set; }
 
+        [Parameter(ParameterSetName = "ExplicitIdentityParameterSet", Mandatory = false)]
+        [Parameter(ParameterSetName = "DefaultParameterSet", Mandatory = false)]//TODO: set sets as string constants?
+        public SwitchParameter EncryptionAtHost { get; set; } = false;
```

Comment:
We can leave out the parameter set names, since these are the only two parameter sets.